OZON.ru.js

```diff
@@ -406,24 +131,18 @@ var testCases = [
 						"creatorType": "author"
 					}
 				],
-				"notes": [],
-				"tags": [],
-				"seeAlso": [],
-				"attachments": [
-					{
-						"title": "OZON.ru Link",
-						"mimeType": "text/html",
-						"snapshot": false
-					}
```

Comment:
In absence of a link back to OZON.ru in the URL field, we sometimes attach links to items as per above. If you think it is useful for the user to be able to get back to OZON.ru from within Zotero, you can include that as well.